sonar-lits-plugin/src/test/java/com/sonarsource/lits/IssuesCheckerTest.java

```diff
@@ -182,7 +182,7 @@ public void should_fail_when_missing_resources() {
       checker.save();
       fail("Expected exception");
     } catch (MessageException e) {
-      assertThat(e.getMessage()).isEqualTo("Missing resources: missing_resource");
+      assertThat(e.getMessage()).isEqualTo("Files listed in Expected directory were not analyzed: missing_resource");
```

Comment:
sorry for realzing now, could you please add a test with multiple missing resources, to test that the concatenation works?